src/core/components/forms/checkboxCard/checkboxCard.test.tsx

```diff
@@ -62,4 +62,10 @@ describe('<CheckboxCard /> component', () => {
         render(createTestComponent({ checked }));
         expect(screen.getByTestId(IconType.CHECKBOX_INDETERMINATE)).toBeVisible();
     });
+
+    it('renders the children when checkbox is checked', () => {
```

Comment:
Can we also have a test to verify that the `children` property is instead hidden when the component is not checked?

Reply:
So it seems jest is not playing nice with tailwind and doesn't recognise the hidden and block classes, so for now I have left this test out